### PR TITLE
Update GettingStartedGuide.md.

### PR DESCRIPTION
**Pull request by iriza7**
* Update GettingStartedGuide.md.

* Fix incorrect text and typo.

Relates-To: OLPEDGE-1118.

Signed-off-by: Bautista, Iryna <ext-iryna.bautista@here.com>

### CODE REVIEW

File: docs/GettingStartedGuide.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 ## <a name="prerequisite"></a> OLP Prerequisite Knowledge
 
-The use of HERE OLP SDK requires a basic understanding of the following OLP-related concepts:
+To use the HERE Open Location Platform (OLP) SDK for TypeScript, you need to understand the following concepts related to OLP:
 
-- [OLP Catalog](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/catalogs.html)
-- [OLP Layers](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/layers.html)
-- [OLP Partitions](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/partitions.html)
-- [HERE Resource Name (HRN)](https://developer.here.com/olp/documentation/data-user-guide/shared_content/topics/olp/concepts/hrn.html)
+- [Catalog](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/catalogs.html)
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]:
```suggestion
- [Catalogs](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/catalogs.html)
```

File: docs/GettingStartedGuide.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+- [Layers](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/layers.html)
+- [Partitions](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/partitions.html)
+- [HERE Resource Names (HRNs)](https://developer.here.com/olp/documentation/data-user-guide/shared_content/topics/olp/concepts/hrn.html)
 
 For more details, see [HERE OLP Data User Guide](https://developer.here.com/olp/documentation/data-user-guide/index.html).
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]:
```suggestion
For more details, see the [HERE OLP Data User Guide](https://developer.here.com/olp/documentation/data-user-guide/index.html).
```

File: docs/GettingStartedGuide.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
 
-## <a name="credentials"></a> Getting Credentials
+## <a name="credentials"></a> Get Credentials
 
 Before any catalog or service request to the [HERE Open Location Platform](https://platform.here.com), you must obtain the OLP authentication and authorization credentials.
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]:
```suggestion
To create catalog or service requests to the [HERE Open Location Platform](https://platform.here.com), you need to have the OLP authentication and authorization credentials.
```

File: docs/GettingStartedGuide.md [hunk 4]
```diff
@@ -31,30 +31,30 @@ The following two authentication approaches are available within your applicatio
 
 > **Note**: to obtain your **platform credentials**, create a new application through the [Apps & Keys](https://platform.here.com/profile/apps-and-keys) page. After creating your application, click **Create A Key** to download these credentials. For more information on obtaining platform credentials, see [Manage Apps](https://developer.here.com/olp/documentation/access-control/user-guide/topics/manage-apps.html) section of the [Teams and Permissions User Guide](https://developer.here.com/olp/documentation/access-control/user-guide/index.html).
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]:
```suggestion
> **Note**: To obtain your **platform credentials**, create an application on the [Apps & Keys](https://platform.here.com/profile/apps-and-keys) page. When the application is created, to download the credentials, click **Create A Key**. For more information on how to get platform credentials, see the [Manage Apps](https://developer.here.com/olp/documentation/access-control/user-guide/topics/manage-apps.html) section of the [Teams and Permissions User Guide](https://developer.here.com/olp/documentation/access-control/user-guide/index.html).
```

File: docs/GettingStartedGuide.md [hunk 5]
```diff
@@ -31,30 +31,30 @@ The following two authentication approaches are available within your applicatio
...
 
-2. Using your **app access key id** and **secret access key**, which you can find on platform's [Apps & Keys](https://platform.here.com/profile/apps-and-keys) page. For more details on how to create your authentication and authorization client for retrieving HERE tokens, see [Authentication and Authorization Developer's Guide](https://developer.here.com/olp/documentation/access_control/topics/introduction.html).
+2. Using your **access key id** and **access key secret**, that you can find on the [Apps & Keys](https://platform.here.com/profile/apps-and-keys) page. For more details on how to create your authentication and authorization client for retrieving HERE tokens, see the [Authentication and Authorization Developer's Guide](https://developer.here.com/olp/documentation/access_control/topics/introduction.html).
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]:
```suggestion
2. Using your **access key id** and **access key secret** that you can find on the [Apps & Keys](https://platform.here.com/profile/apps-and-keys) page. For more details on how to create your authentication and authorization client for retrieving HERE tokens, see the [Authentication and Authorization Developer's Guide](https://developer.here.com/olp/documentation/access_control/topics/introduction.html).
```

File: docs/GettingStartedGuide.md [hunk 6]
```diff
@@ -31,30 +31,30 @@ The following two authentication approaches are available within your applicatio
...
+[Volatile Layer Read example](./examples/nodejs-read-volatile-layer.md) shows how to retrieve layer metadata, partition metadata, and partition data from a volatile layer using the OLP SDK for TypeScript.
 
 ## <a name="catalog-client"></a> CatalogClient
 
-The main entry point to the OLP is `CatalogClient`. This class provides a high-level interface for retrieval of `OLP` catalog data and configuration, and defines the following operations:
+The main entry point to OLP is `CatalogClient`. This class provides a high-level interface to retrieve `OLP` catalog data and configuration, and defines the following operations:
 
-- `getLatestVersion`: Fetches the latest version of the catalog
-- `getVersions`: Fetches a tile from this layer asynchronously
+- `getLatestVersion`: Fetches the latest version of the catalog.
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]:
```suggestion
- `getLatestVersion`: fetches the latest version of the catalog.
```

File: docs/GettingStartedGuide.md [hunk 7]
```diff
@@ -31,30 +31,30 @@ The following two authentication approaches are available within your applicatio
...
+- `getVersions`: Fetches a tile from this layer asynchronously.
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]:
```suggestion
- `getVersions`: fetches a tile from this layer asynchronously.
```

**Comment by iriza7** [comment]
All fixed now

**Pull request merged**